scapy/utils.py

```diff
@@ -1657,23 +1669,20 @@ def _read_packet(self, size=MTU):  # type: ignore
 
     def _read_options(self, options):
         # type: (bytes) -> Dict[str, Any]
-        """Section Header Block"""
-        opts = self.default_options.copy()  # type: Dict[str, Any]
+        opts = dict()
         while len(options) >= 4:
             code, length = struct.unpack(self.endian + "HH", options[:4])
             # PCAP Next Generation (pcapng) Capture File Format
             # 4.2. - Interface Description Block
             # http://xml2rfc.tools.ietf.org/cgi-bin/xml2rfc.cgi?url=https://raw.githubusercontent.com/pcapng/pcapng/master/draft-tuexen-opsawg-pcapng.xml&modeAsFormat=html/ascii&type=ascii#rfc.section.4.2
-            if code == 9 and length == 1 and len(options) >= 5:
-                tsresol = orb(options[4])
-                opts["tsresol"] = (2 if tsresol & 128 else 10) ** (
-                    tsresol & 127
-                )
-            if code == 1 and length >= 1 and 4 + length < len(options):
-                opts["comment"] = options[4:4 + length]
+
+            if code != 0 and 4 + length < len(options):
+                opts[code] = options[4:4 + length]
+
             if code == 0:
                 if length != 0:
-                    warning("PcapNg: invalid option length %d for end-of-option" % length)  # noqa: E501
+                    warning("PcapNg: invalid option "
+                            "length %d for end-of-option" % length)  # noqa: E501
```

Comment:
Is noqa still useful here?

Reply:
I think it can be removed or the change undone.